File: java/parser/src/main/java/gov/lbl/parser/parser/Rel_opVisitor.java
package gov.lbl.parser.parser;

import gov.lbl.antlr4.visitor.modelicaBaseVisitor;
import gov.lbl.antlr4.visitor.modelicaParser;

public class Rel_opVisitor  extends modelicaBaseVisitor<String> {
    @Override
    public String visitRel_op(modelicaParser.Rel_opContext ctx) {
        String rel_op = ctx == null ? "" : ctx.getText();
        return rel_op;
    }
}

File: jsParser/parser/Rel_opVisitor.js
const modelicaVisitor = require('../antlrFiles/modelicaVisitor').modelicaVisitor

class Rel_opVisitor {
  constructor () {
    modelicaVisitor.call(this)
    return this
  }

  visitRel_op (ctx) {
    const rel_op = ctx ? ctx.getText() : ''
    return rel_op
  }
};

Rel_opVisitor.prototype = Object.create(modelicaVisitor.prototype)

exports.visitRel_op = this.visitRel_op
exports.Rel_opVisitor = Rel_opVisitor
